spec/requests/api/v1/article_spec.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.describe 'List articles', type: :request do
+  let(:headers) { { HTTP_ACCEPT: 'application/json' } }
+
+  context 'with valid articles' do
+    it 'returns a list of articles' do
+      get '/api/v1/articles', params: { title: 'Trump Impeachment',
```

Comment:
this test is not thorough enough, please take a look at the performance data index request spec from the cooper challenge